### ISSUE TITLE
Weird black bars on hover for ggplotly bubble chart conversion

### ISSUE THREAD

**Issue by corinne-riddell** [ropensci/plotly#443]
The Plotly version is also missing the bubbles.

Plotly-ized version after calling ggplotly() (notice black bars on hover):
![image](https://cloud.githubusercontent.com/assets/12090129/12699044/ab30c4d0-c77b-11e5-880c-488f02271160.png)

Original ggplot2 version:

![image](https://cloud.githubusercontent.com/assets/12090129/12699051/da9df3aa-c77b-11e5-854e-70e9d1bcc04e.png)

**Comment by corinne-riddell** [comment]
Updating the plotly R library resolved the issue with the bubbles not showing. The plotly-ized version still has some problems displaying the hover text (it seems to be concatenating all the labels into one big label...)

![screen shot 2016-01-30 at 6 37 30 pm](https://cloud.githubusercontent.com/assets/12090129/12699183/9de54ea4-c780-11e5-8ccd-6541744e6a01.png)

**Comment by cpsievert** [comment]
Thanks @corinne-riddell, and cool plot! Would it be possible to share the data/code for this example?

**Comment by corinne-riddell** [comment]
Hey! I'll copy the r code below. This will remake the ggplot and the plotly-ized version. Lemme know if there are any problems. Thank you :)

```
library(ggplot2)
library(scales)
library(sp)
library(plotly)

csdat <- read.csv("http://corinne-riddell.github.io/datasets/CSglobal.csv")
gdpdat <- read.csv("http://corinne-riddell.github.io/datasets/yearlyGDPbyCountry.csv")
moredat <- read.csv("http://corinne-riddell.github.io/datasets/otherCountryInfo.csv")
names(moredat)[2] <- "CountryCode"

dat <- merge(csdat, gdpdat, by="CountryCode", all.Y=F, all.X=T)
dat <- merge(dat, moredat, by="CountryCode", all.Y=F, all.X=T)
dat$CS_Rate2 <- dat$CS_Rate/100

loessmod <- loess(dat$CS_Rate2 ~ log(dat$X2006), weights=dat$Births_Per_1000)
loess_preds <- predict(loessmod, dat, se=T)
dat$pred_smooth <- loess_preds$fit
dat$smooth_ucl<- loess_preds$fit + 2*loess_preds$se.fit
dat$smooth_lcl<- loess_preds$fit - 2*loess_preds$se.fit

a <- ggplot(dat=dat, aes(y=CS_Rate2, x=log(X2006), label=Label)) +
  geom_ribbon(aes(ymin = smooth_lcl, ymax = smooth_ucl), fill = "#cccccc") + 
  geom_line(aes(y = pred_smooth)) + 
  geom_point(col="white", aes(fill=IncomeGroup, size=Births_Per_1000), shape=21, alpha=0.8) + 
  scale_size_area(max_size=50, guide="none") +
  geom_text(size=4) + #scale_fill_discrete(name="Income Group") +
  #scale_fill_manual(name="Income Group") +
  guides(fill=guide_legend(override.aes = list(size=5))) + 
  theme_minimal(base_size = 15) + 
  theme(legend.position="bottom")  +
  scale_y_continuous(name="Cesarean delivery rate", labels=percent) + 
  scale_x_continuous(name="log of per capita GDP in 2006 (2015 US$)")  #scale_size(guide='none') +

a

(gg <- ggplotly(a))

gg```

**Comment by cpsievert** [comment]
This should be fixed in f1900cc3cd464408a60ffadfe3e95460747430e0. Let us know if you still have problems.
